tool/tsh/tsh.go

```diff
@@ -1360,22 +1361,31 @@ func printNodesAsText(nodes []types.Server, verbose bool) {
 	// In normal mode chunk the labels and print two per line and allow multiple
 	// lines per node.
 	case false:
-		t = asciitable.MakeTable([]string{"Node Name", "Address", "Labels"})
+		var rows [][]string
 		for _, n := range nodes {
-			labelChunks := chunkLabels(n.GetAllLabels(), 2)
-			for i, v := range labelChunks {
-				if i == 0 {
-					t.AddRow([]string{n.GetHostname(), getAddr(n), strings.Join(v, ", ")})
-				} else {
-					t.AddRow([]string{"", "", strings.Join(v, ", ")})
-				}
-			}
+			rows = append(rows,
+				[]string{n.GetHostname(), getAddr(n), sortedLabels(n.GetAllLabels())})
 		}
+		t = makeTableWithTruncatedFinalColumn([]string{"Node Name", "Address", "Labels"}, rows)
 	}
-
 	fmt.Println(t.AsBuffer().String())
 }
 
+func sortedLabels(labels map[string]string) string {
+	var namespaced []string
+	var result []string
+	for key, val := range labels {
+		if strings.Contains(key, "/") {
```

Comment:
I would consider treating the `teleport.dev/` namespace as a special case and ensuring those are always last.

It's entirely possible for users to put `/` in their own labels, and we should consider them more important to show than the ones that teleport applies if there is a space constraint.